docs/MAINTAIN.md

```diff
@@ -0,0 +1,16 @@
+# Supporting new families
+
+## Write device file
+
+a file <family>.cmake must be added to the folder cmake/stm32
+This file containes the differents devices and the regex used to parse them.
+It also give information on the RAM and CCRAM available
+
+## Add family to list of know families
+
+Update the list `STM32_SUPPORTED_FAMILIES_LONG_NAME` located in the cmake/stm32/common.cmake file
+
+
+## Add family to CI
+
+Update the cmake.yml workflow file to include the new family
```

Comment:
I'm not sure we already automated. Maintainer must add the list of HAL and LL somewhere

Reply:
HE have some basics tests yes. It does not pass for now, since I just started making the changes

Reply:
finHAL has been implemented. Please ignore this comment